### PR TITLE
17577 dashboard va benefits cta

### PR DESCRIPTION
**Pull request by tlei123**
## Description
[17577](/department-of-veterans-affairs/vets.gov-team/issues/17577) - Add CTA button to Dashboard | Find VA Benefits | Family section, linking to Family Member Benefits landing page (Headline is "VA benefits for spouses, dependents, survivors, and family caregivers):
**Button copy** —  "View all related benefits"
**Link URL** — /family-member-benefits/

## Testing done
Verified locally, new button is linking to correct landing page.  (No logic-/functional-code changes -- basically just a new link.)

## Screenshots
Dashboard-Find VA Benefits-Family and Caregiver

## Acceptance criteria
- [ ] New CTA with correct label displayed on Dashboard page under Family and Caregiver Benefits.
- [ ] CTA-click navigates to correct landing page.

## Definition of done
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/preferences/helpers.jsx [hunk 1]
```diff
@@ -189,6 +189,11 @@ export const benefitChoices = [
     gaTag: 'Family',
     introduction:
       'If you’re the family member of a Veteran or Servicemember, you may qualify for benefits yourself. If you’re a caregiver for a Veteran with service-connected disabilities, you may qualify for additional benefits and support for yourself and the Veteran you’re caring for.',
+    cta: {
+      link: '/family-member-benefits/',
+      text: 'View all related benefits',
+      gaTag: 'View all related benefits',
```
Comment (erikphansen) [inline src/applications/personalization/preferences/helpers.jsx]: Should this be kebab-case?

Comment (tlei123) [reply src/applications/personalization/preferences/helpers.jsx]: The case is the same as the other pre-existing blocks -- I just copied an existing cta block and pasted it into the Family block for this new CTA button.
That said, I can definitely do a file-global find-n-replace if you think the case should be different.
Feel free to either Resolve the convo (let it be) or Comment further (change all key-instances).

Comment (erikphansen) [reply src/applications/personalization/preferences/helpers.jsx]: which `cta` block did you copy? I see a bunch of `cta.gaTag`s being kebab-case (line 176, 138, 117, etc)

Comment (tlei123) [reply src/applications/personalization/preferences/helpers.jsx]: Strange!
Can't remember which one I copied from, BUT I just double-checked using this PR's View File (/department-of-veterans-affairs/vets-website/blob/a100693cf5f8f97f56d50356fd4afd5fc9e9190f/src/applications/personalization/preferences/helpers.jsx) and I'm not seeing what you saw -- all the GA Tag keys are 'gaTag' there.

Comment (erikphansen) [reply src/applications/personalization/preferences/helpers.jsx]: LOL, total miscommunication here. I meant the _value_ of the `gaTag` should be `view-all-related-benefits`

Comment (tlei123) [reply src/applications/personalization/preferences/helpers.jsx]: ROFL!  I'll kebab the value.
Leaving this convo unresolved 'til my fix is pushed up.

Comment (tlei123) [reply src/applications/personalization/preferences/helpers.jsx]: gaTag value kebab-case fix pushed up.

**Review by erikphansen — APPROVED** [review]
I'll try to be less vague in the future!

**Pull request merged**